<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanSearchDialog</class>
 <widget class="QDialog" name="HumanSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>942</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>942</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>人事档案查询</string>
  </property>
  <widget class="QGroupBox" name="groupBox_staff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>教工信息</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_name"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_native">
       <property name="text">
        <string>籍贯：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_native"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_age">
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_age">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,4">
     <item>
      <widget class="QLabel" name="label_sex">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_sex">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_nation">
       <property name="text">
        <string>民族：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_nation">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_marriage">
       <property name="text">
        <string>婚姻状况：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_marriage">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>未婚</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>已婚</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_technical">
       <property name="text">
        <string>职称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_technical">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_post">
       <property name="text">
        <string>职务：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_post">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,5">
     <item>
      <widget class="QLabel" name="label_education">
       <property name="text">
        <string>文化程度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_education">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,5">
     <item>
      <widget class="QLabel" name="label_political">
       <property name="text">
        <string>政治面貌：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_political">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_unit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>单位信息</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,5">
     <item>
      <widget class="QLabel" name="label_unit">
       <property name="text">
        <string>院系部门：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_unit">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,5">
     <item>
      <widget class="QLabel" name="label_position">
       <property name="text">
        <string>教工岗位：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_position">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>120</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="list_search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>921</width>
     <height>221</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>HumanSearchDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
